examples/basic.rs

```diff
@@ -222,50 +227,32 @@ fn board_pos_to_world_pos(pos: &UVec2) -> Vec2 {
 struct Selection(Option<Entity>);
 
 fn input(
-    windows: Res<Windows>,
+    window_query: Query<&Window, With<PrimaryWindow>>,
     mut selection: ResMut<Selection>,
-    mut button_events: EventReader<MouseButtonInput>,
+    mut button_events: Res<Input<MouseButton>>,
```

Comment:
Why this change? Afaik button events should still work

Reply:
I'm still a bit new to rust and pretty sure I was fighting complier errors, and I found `Res<Input<MouseEvent>>` got what I needed. But I changed it back, and it should be good now.